src/class-parsely.php

```diff
@@ -397,12 +398,6 @@ public function insert_page_header_metadata(): void {
 	 *              By default,only 'publish' is allowed.
 	 */
 	public static function post_has_trackable_status( $post ): bool {
-		static $cache = array();
-		$post_id      = is_int( $post ) ? $post : $post->ID;
-		if ( isset( $cache[ $post_id ] ) ) {
-			return $cache[ $post_id ];
-		}
```

Comment:
@vaurdan, I need your opinion on this one.

This function uses caching, which I removed in the moment. I'd guess caching assumed that post status won't change while the different functions of the request are running. For one, I'm not 100% certain that the assumption is correct, and second this actually made some of the new integration tests fail (which is how I became aware of the caching).

The tests can be tinkered to work with caching, so let me know if you think we should restore it.

Reply:
It is possible for the post status to change while the this request is running, if a post is published, for example, while the request runs. Seems highly unlikely to happen, but it can. 

As for the cache here, I don't really see a point. I guess it could make sense if either `post_has_trackable_status` or even `call_update_metadata_endpoint` would be called within a loop and would be called multiple times for the same post. But from a quick inspection it looks like it should only be called once per post, so there aren't any real gains on having a caching mechanism here, IMO.

Reply:
Thanks. Let's go without the caching here.